src/hotspot/share/oops/symbol.cpp

```diff
@@ -390,11 +389,10 @@ void Symbol::print() const { print_on(tty); }
 // The print_value functions are present in all builds, to support the
 // disassembler and error reporting.
 void Symbol::print_value_on(outputStream* st) const {
-  st->print("'");
-  for (int i = 0; i < utf8_length(); i++) {
-    st->print("%c", char_at(i));
-  }
-  st->print("'");
+  st->write("'", 1);
```

Comment:
I haven't seen code using the 'write' function of ostream, but usually print_raw.  Can you use that instead?  print_raw just calls write.